### PR TITLE
company wide policy cleanup

### PR DESCRIPTION
**Pull request by peterlangmar**
no real change, just made it easier to read / understand

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -381,13 +381,13 @@ To account any kinds of financial liabilities, we use [Splitwise](https://www.sp
 
 
 
-## Domain: other Company-Wide policies (default for all roles and circles)
+## Domain: other Company-Wide policies (mandatory by default for all roles and circles)
 
 ##### Policy: Calendar and meetings
-Have your calendar email address at ContactSheet.
-Share your private calendar free-busy status publicly (or with everyone in the ContactSheet). Keep your private calendar free-busy status for work hours up to date. Pay extra attention to mark full-day events as "busy" if you're not available.
+Share your private calendar free-busy status publicly (or with everyone in the ContactSheet). Keep your private calendar free-busy status for work hours up to date. Pay extra attention to mark full-day events as "busy" if you're not available. Have your calendar email address at "Personal data + Contact" sheet.
```
Comment (cassus) [inline README.md]: Would be nice to have a link to the ContactSheet.

File: README.md [hunk 2]
```diff
@@ -439,9 +439,9 @@ Time boxing is important, to not spend too much time on standupping
 ##### Policy: Time tracking
 We should track the time dedicated to each role, even internal roles. Preferably use one, company wide tool.
 
-##### Policy: Transparency levels (mandatory for all roles and circles)
+##### Policy: Transparency levels
 Public:
- - [Governance](https://github.com/Brickflow/lab-governance) records
+ - [Governance](https://github.com/lab-coop/lab-governance) records
```
Comment (cassus) [inline README.md]: :+1:

File: README.md [hunk 3]
```diff
@@ -381,13 +381,13 @@ To account any kinds of financial liabilities, we use [Splitwise](https://www.sp
 
 
 
-## Domain: other Company-Wide policies (default for all roles and circles)
+## Domain: other Company-Wide policies (mandatory by default for all roles and circles)
```
Comment (cassus) [inline README.md]: :+1:

File: README.md (hunk 1 above)
Comment (peterlangmar) [inline README.md]: good idea! I'm adding it to the sheet

**Pull request merged**